.github/workflows/cursor-rule-research-agent.yml

```
@@ -0,0 +1,168 @@
+            --data "$JSON_PAYLOAD")
+
+          AGENT_URL=$(echo "$RESPONSE" | jq -r '.target.url')
+          echo "agent_url=$AGENT_URL" >> $GITHUB_OUTPUT
+
+      - name: Comment on issue with agent link
+        if: steps.check_research.outputs.has_research != 'true'
+        run: |
+          URL='${{ steps.launch.outputs.agent_url }}'
+          gh api repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/comments \
+            -f body="🔍 Launched Cursor Research Agent to check for existing ESLint rules. View agent: $URL"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)